crowbar_framework/app/models/ceph_service.rb

```diff
@@ -142,7 +143,10 @@ def create_proposal
       n.intended_role == "controller" or osd_nodes.include? n
     end.first
     if mds_node.nil?
-      mds_node = select_nodes_for_role(nodes, "ceph-mds", "controller").first
+      mds_node = select_nodes_for_role(nodes, "ceph-mds", "controller")
+      mds_node = mds_node.select do |n|
+        CrowbarPacemakerHelper.is_cluster_founder?(n)
```

Comment:
This actually not going to work because in our CI we only have one node intended as a controller (which is usually the founder too) and so this case will never be hit.